kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/RollingUpdateUserTaskServiceIntegrationTest.java

```diff
@@ -64,7 +64,7 @@ public void testGetTaskInputAndOutputWithAlias() throws Exception {
             ProcessInstance processInstance = processClient.getProcessInstance(CONTAINER_ALIAS, pid);
             assertThat(processInstance).isNotNull();
 
-            List<TaskSummary> taskList = taskClient.findTasksAssignedAsBusinessAdministrator(USER_YODA, 0, 10);
+            List<TaskSummary> taskList = taskClient.findTasksAssignedAsBusinessAdministrator(USER_ADMINISTRATOR, 0, 10);
```

Comment:
Why this change?

Reply:
this is a faulty test.  before it was working because because the bypass was on false. But when I changed it the the administrator user was the right one to set.